src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -2602,6 +2602,15 @@ ItemDelete.Quality = 3
 
 ItemDelete.ItemLevel = 80
 
+#
+#
+#    ItemDelete.KeepDays
+#        Description: Time (in days)
+#        Default:     30 - (Enabled)
+#                     0  - (Disabled, Don't delete any items)
+
+ItemDelete.KeepDays = 0
```

Comment:
You say the default is 30, but you set it to 0 🤔

Reply:
😨